runtime/eventlog.c

```diff
@@ -132,32 +132,23 @@ static void setup_evbuf()
 
 static void setup_eventlog_file()
 {
-  CAMLparam0();
-  char_os *filename = NULL;
-  char_os *eventlog_filename;
-  CAMLlocal1 (tmp);
+  char_os output_file[4096];
+  int output_file_len = sizeof(output_file)/sizeof(char_os);
```

Comment:
FWIW I would do this is a definition before the function:
```c
#define OUTPUT_FILE_LEN 4096
```
and use that in place of `output_file_len` and then do
```c
#undef OUTPUT_FILE_LEN
```
at the end of the function - that seems better than having to worry about the `sizeof`